cli/commands/seek.py

```diff
@@ -0,0 +1,77 @@
+import click
+import re
+
+from ..utils import Spotify
+from ..utils.exceptions import *
+
+
+@click.command(options_metavar='[<options>]')
+@click.option(
+    '-f', '--forward', is_flag=True,
+    help='Increment playback position'
+)
+@click.option(
+    '-r', '--rewind', is_flag=True,
+    help='Decrement playback position'
+)
+@click.argument(
+    'position', type=str,
+    metavar='<position>'
+)
+def seek(forward, rewind, position):
+    """Seek to time (in seconds if no unit is mentioned) in current track.
```

Comment:
i think wording this like "Seek to time (default unit: seconds) in the current track." would be nicer